[plugin.video.tagesschau] 2.3.0

christianprasch opened the pull request:
### Description
This is the Tagesschau addon from J.Schumacher and H.Saul updated to Kodi 19 (Matrix)
### Checklist:
<!--- Go over all the following points, and put an `x` in all the boxes that apply like this: [X] -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.
- If you see no activity on your PR after a week (so at least one weekend has passed) then please go to the #kodi-dev freenode IRC channel to reach out to the team

basrieter commented:
I would prefer this PR to go via a PR in the original add-on repository of plugin.video.tagesschau: https://github.com/joerns/xbmc-plugin.video.tagesschau/issues.

@joerns Could you let us know if you still accept contributions via your own GitHub or that are OK with us accepting this update version of your add-on?

basrieter commented on plugin.video.tagesschau/addon.xml:
```diff
@@ -0,0 +1,63 @@
+  </extension>
+  <extension point="xbmc.addon.metadata">
+    <language>de</language>
+    <platform>all</platform>
```
Please add a proper license tag. Example: <license>GPL-3.0-only</license>

basrieter replied:
Also add a proper <source></source> tag that points to the correct github.

basrieter commented on plugin.video.tagesschau/addon.xml:
```diff
@@ -0,0 +1,63 @@
+      <screenshot>resources/assets/screenshot_3.png</screenshot>
+    </assets>
+    <news>
+version 2.3.0
```
Try to keep this a bit more to the most recent ones.

basrieter commented on plugin.video.tagesschau/libs/tagesschau_json_api.py:
```diff
@@ -0,0 +1,412 @@
+        handle = urllib.request.urlopen("http://www.tagesschau.de/api/multimedia/sendung/letztesendungen100~_week-true.json")
+        return json.loads(handle.read())
+        
+if __name__ == "__main__":
```
is this needed?

basrieter reviewed (changes requested):
Please have a quick look at my suggestions.

joerns commented:
> I would prefer this PR to go via a PR in the original add-on repository of plugin.video.tagesschau: https://github.com/joerns/xbmc-plugin.video.tagesschau/issues.
> 
> @joerns Could you let us know if you still accept contributions via your own GitHub or that are OK with us accepting this update version of your add-on?

Hi @basrieter, thanks for checking. @christianprasch did check with me beforehand. I am no longer supporting this plugin, I would be happy if Christian takes over. PRs should then go via his forked repo. Cheers

christianprasch commented:
Hi @basrieter, i have implemented the requested changes and updated the PR as described in "HOW-TO:Update a pull request to an official addon repository on GitHub". Hopefully i have done it in the correct way, i am completely new to this workflow.
Please let me know when i missed something.
Regards, Christian

basrieter commented:
All merged. Thanks for your contribution.
